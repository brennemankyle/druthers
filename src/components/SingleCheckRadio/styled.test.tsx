// import React from "react";
// import renderer from "react-test-renderer";
// import { mockStyles, MockSvg } from "../../mocks";
// import { CheckBox, Radio, Switch } from "./styled";

// const props = {
//   ...mockStyles,
//   name: "test",
//   value: "value",
//   className: "test",
//   label: "test label",
//   disabled: false,
//   toggle: false,
//   checked: false,
//   multiple: false,
//   svg_Checkmark: MockSvg
// };

// describe("CheckBox", () => {
//   it("styles", () => {
//     const wrapper = renderer.create(<CheckBox {...props} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("disabled = true", () => {
//     const wrapper = renderer.create(<CheckBox {...props} disabled={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("checked = true", () => {
//     const wrapper = renderer.create(<CheckBox {...props} checked={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });
// });

// describe("Radio", () => {
//   it("styles", () => {
//     const wrapper = renderer.create(<Radio {...props} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("disabled = true", () => {
//     const wrapper = renderer.create(<Radio {...props} disabled={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("checked = true", () => {
//     const wrapper = renderer.create(<Radio {...props} checked={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });
// });

// describe("Switch", () => {
//   it("styles", () => {
//     const wrapper = renderer.create(<Switch {...props} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("disabled = true", () => {
//     const wrapper = renderer.create(<Switch {...props} disabled={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });

//   it("checked = true", () => {
//     const wrapper = renderer.create(<Switch {...props} checked={true} />);

//     expect(wrapper).toMatchSnapshot();
//   });
// });
