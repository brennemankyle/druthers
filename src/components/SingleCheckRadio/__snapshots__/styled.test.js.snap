// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox checked = true 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  border-radius: .2em;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: white;
}

.emotion-0 .check-radio-display svg {
  fill: black;
  width: 1em;
  height: 1em;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={true}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`CheckBox disabled = true 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: not-allowed;
}

.emotion-0.emotion-0 .check-radio-display {
  border-color: #ECECEC;
}

.emotion-0 .check-radio-display {
  border-radius: .2em;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: #ECECEC;
}

.emotion-0 .check-radio-display svg {
  fill: black;
  width: 1em;
  height: 1em;
  visibility: hidden;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={true}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`CheckBox styles 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  border-radius: .2em;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: white;
}

.emotion-0 .check-radio-display svg {
  fill: black;
  width: 1em;
  height: 1em;
  visibility: hidden;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Radio checked = true 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  border-radius: 100%;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  background-size: 0.1em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: white;
  background-color: #C3C3C3;
}

.emotion-0 .check-radio-display svg {
  visibility: hidden;
  width: 1em;
  height: 1em;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={true}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Radio disabled = true 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: not-allowed;
}

.emotion-0.emotion-0 .check-radio-display {
  border-color: #ECECEC;
}

.emotion-0 .check-radio-display {
  border-radius: 100%;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  background-size: 0.1em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: #ECECEC;
}

.emotion-0 .check-radio-display svg {
  visibility: hidden;
  width: 1em;
  height: 1em;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={true}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Radio styles 1`] = `
.emotion-0 {
  margin-right: 1em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  border-radius: 100%;
  margin-right: .2em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  background-size: 0.1em;
  border: 2px solid #C3C3C3;
  width: 1em;
  height: 1em;
  background-color: white;
}

.emotion-0 .check-radio-display svg {
  visibility: hidden;
  width: 1em;
  height: 1em;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Switch checked = true 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  margin-right: .2em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 1em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: calc(1em * 2);
  height: 1em;
  background-color: white;
}

.emotion-0 .check-radio-display svg {
  width: 1em;
  height: 1em;
  border-radius: 100%;
  margin-left: 1em;
  background-color: black;
  fill-opacity: 0;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={true}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Switch disabled = true 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: not-allowed;
}

.emotion-0.emotion-0 .check-radio-display {
  border-color: #ECECEC;
}

.emotion-0 .check-radio-display {
  margin-right: .2em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 1em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: calc(1em * 2);
  height: 1em;
  background-color: #ECECEC;
}

.emotion-0 .check-radio-display svg {
  width: 1em;
  height: 1em;
  border-radius: 100%;
  background-color: #C3C3C3;
  fill-opacity: 0;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={true}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;

exports[`Switch styles 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  cursor: pointer;
}

.emotion-0:hover .check-radio-display {
  border-color: black;
}

.emotion-0.focus .check-radio-display {
  border-color: black;
}

.emotion-0 .check-radio-display {
  margin-right: .2em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 1em;
  padding-top: .2em;
  padding-bottom: .2em;
  padding-right: .2em;
  padding-left: .2em;
  border: 2px solid #C3C3C3;
  width: calc(1em * 2);
  height: 1em;
  background-color: white;
}

.emotion-0 .check-radio-display svg {
  width: 1em;
  height: 1em;
  border-radius: 100%;
  background-color: #C3C3C3;
  fill-opacity: 0;
}

.emotion-0 input {
  opacity: 0;
  width: 0;
}

<label
  className="test emotion-0 "
>
  <input
    checked={false}
    disabled={false}
    name="test"
    onBlur={[Function]}
    onFocus={[Function]}
    type="radio"
    value="value"
  />
  <div
    className="check-radio-display"
  >
    <svg>
      <path
        d="M0 0L5 5Z"
      />
    </svg>
  </div>
  test label
</label>
`;
